packages/synapse-react-client/src/components/BasePortalCard/ChallengePortalCard/ChallengePortalCard.tsx

```diff
@@ -0,0 +1,220 @@
+import { parseEntityIdAndVersionFromSqlStatement } from '@/utils/functions'
+import BasePortalCard from '../BasePortalCard'
+import { QueryBundleRequest, Row } from '@sage-bionetworks/synapse-types'
+import { SynapseConstants } from '@/utils'
+import useGetQueryResultBundle from '@/synapse-queries/entity/useGetQueryResultBundle'
+import { getFieldIndex } from '@/utils/functions/queryUtils'
+import { Chip, Box, Button, Stack, Typography } from '@mui/material'
+import styles from './ChallengePortalCard.module.scss'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import { useImageUrl } from '@/utils/hooks/useImageUrlUtils'
+import { Close } from '@mui/icons-material'
+import { ReactComponent as CircleImage } from '../../../assets/illustrations/circle.svg'
+import { ReactComponent as LongLineImage } from '../../../assets/illustrations/longLine.svg'
+import { ReactComponent as ShortLineImage } from '../../../assets/illustrations/shortLine.svg'
+import { ReactComponent as TriangleImage } from '../../../assets/illustrations/triangle.svg'
+import { ReactComponent as DoubleLineImage } from '../../../assets/illustrations/doubleLine.svg'
+import { hashCode } from '@/utils/functions/StringUtils'
+
+type ChallengePortalCardProps = {
+  sql: string
+  borderRadius?: number
+}
+
+type ChallengePortalCardRowProps = {
+  row: Row
+  entityId: string | undefined
+  indices: Record<string, number>
+  sql: string
+  borderRadius?: number
+}
+
+const colors = ['#F8CC7D', '#BFE8F4', '#CEFBDD']
+
+const getChicletColors = (badges: string[]) => {
+  let lastColorIndex = -1
+  return badges.map((badge, i) => {
+    const hash = hashCode(badge)
+    let index = Math.abs(hash + i) % colors.length
+
+    // prevent consecutive badges from being the same color
+    if (index === lastColorIndex && colors.length > 1) {
+      index = (index + 1) % colors.length
+    }
+
+    lastColorIndex = index
+    return colors[index]
+  })
+}
+
+const Chips = ({ badges }: { badges: string[] }) => {
+  const badgeColors = getChicletColors(badges)
```

Comment:
Recommend a consistent term for these things, either badges, chiclets, or chips.  I think MUI calls them Chips, the Design Team calls them chiclets, and Web Engineering calls them chips/chiclets/badges.  
My vote would be to call them all chips to align with MUI, but I'd support any of these terms.
Also probably call the `badges: string[]` prop `values` instead